1. Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.

In [1]:
import numpy as np
import math as math

In [4]:
a_1 = 80 - 1.96 * 16/10
a_2 = 80 + 1.96 * 16/10
print(f'confidence interval {a_1} - {a_2}')

confidence interval 76.864 - 83.136


2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.

In [14]:
b = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
b_aver = np.average(b)
b_aver
b_std = np.std(b, ddof=1)
b_std
b_1 = b_aver - 2.262 * b_std/math.sqrt(10)
b_2 = b_aver + 2.262 * b_std/math.sqrt(10)
print(f'confidence interval {b_1} - {b_2}')

confidence interval 6.267538255912426 - 6.912461744087575


3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

In [15]:
c_z_r = (17.5 - 17) / (4/10)
c_z_r

1.25

1.25 < 1.65, H0 = True

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [22]:
d = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
d_aver = np.average(d)
d_std = np.std(d)
d_t_r = (200 - d_aver) / (d_std/np.sqrt(10))
d_t_r

1.1227217828476797

1.12 < 2.82, H0 = True